config/state_machines.rst

```diff
@@ -49,9 +61,11 @@ List all of your states here. For examples:
            label: Step 2
        transitions:
 
+The first state must be ``start:`` or the system will throw errors when trying to initialize this value.  All other states can be any string as defined by the user.
+
 transitions:
 ~~~~~~~~~~~~
-List of one (or more) values, each is a type: :doc:`state_machine_transitions <state_machine_transitions>`.
+List of one (or more) values, with the applicable settings, which can be found here: :doc:`state_machine_transitions <state_machine_transitions>`.  These move from any state to another state, including backward or back to the first step, when a given event is posted.
```

Comment:
The first line it auto generated. I will move the second sentence into the text of that section so it won't be overwritten.

Reply:
Got it.  Sounds good to me!  Thanks for looking it over.